### PR TITLE
Draft nav

### LINKED ISSUE #25: Navbar and Footer
Create reusable Navbar and Footer components

### PR DESCRIPTION
**Pull request by sampler36**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

- [x] New feature (non-breaking change which adds functionality)

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

- [x] This change requires a documentation update

## Change status

- [x] Complete, tested, ready to review and merge

- [x] Complete, but not tested (may need new tests)

- [x] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:

- [x] My code follows the style guidelines of this project

- [x] I have performed a self-review of my own code

- [x] I have commented my code, particularly in hard-to-understand areas

- [x] I have made corresponding changes to the documentation, if relevant

- [x] I have run the app using my feature and ensured that no functionality is broken

- [x] My changes generate no new warnings

- [x] There are no merge conflicts

RESOLVE #25

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ import { connect } from 'react-redux';
 import { fetchAllUsers } from './store/actions/users';
 import { loginRequest, registerUser, logout } from './store/actions/auth';
 
-import NavBar from './components/NavBar';
-import Logout from './components/Logout';
+import NavBar from './components/layout/navbar/NavBar';
```
Comment (DianaIgnatescu) [inline src/App.js]: Please try to keep the components as flat as possible and avoid nesting. As a followup to that it's better to have components with longer(but descriptive) names than to have multiple separate directories.

File: src/components/layout/login/SignedInLink.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+
+const LoggedIn = () => {
```
Comment (DianaIgnatescu) [inline src/components/layout/login/SignedInLink.js]: Please use conditional rendering

File: src/components/layout/logout/SignedOutLink.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+// import PropTypes from './node_modules/prop-types';
+// import { Redirect } from 'react-router-dom';
+
+const LoggedOut = () => {
```
Comment (DianaIgnatescu) [inline src/components/layout/logout/SignedOutLink.js]: Same as above.

File: package.json [hunk 4]
```diff
@@ -3,7 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^3.9.3",
     "axios": "^0.18.0",
+    "material-ui": "^0.20.2",
```
Comment (ConConRob) [inline package.json]: Why do we need this ?

**Pull request closed without merging**